app/experimenter/reporting/tests/factories.py

```diff
@@ -12,6 +13,7 @@
 
 
 class ReportLogFactory(ReportLogConstants, factory.django.DjangoModelFactory):
```

Comment:
Django's been hassling me about using `timezone.now()` instead of `datetime.now()`, should probably use that so it doens't give you grief later